api/manifest.go

```diff
@@ -25,13 +26,21 @@ func apiManifestHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
+	paths := shared.ParsePathsParam(r)
 	ctx := appengine.NewContext(r)
-	if manifest, err := getManifestForSHA(ctx, sha); err != nil {
+	manifestBytes, err := getManifestForSHA(ctx, sha)
+	if err != nil {
 		http.Error(w, err.Error(), http.StatusNotFound)
-	} else {
-		w.Header().Add("content-type", "application/json")
-		w.Write(manifest)
+		return
 	}
+	w.Header().Add("content-type", "application/json")
```

Comment:
Nit: Although I think Go probably handles the capitalization already, the standard key should be `Content-Type` here.

Reply:
Done.